checkmarx-ltd/ServiceNowCxOne
# Rest API calls added User-Agent header with information plugin name and plugin version

**Changes**
In a Rest API call added User-Agent header with information plugin name and plugin version.

**Test Cases** 
- Go to All  -> Integration  -> Source Instance -> Checkmarx One Application Vulnerability -> update log level -> all
- Run 1st integration
- Open All -> OutBound Http Requests -> Open log -> Requests -> Request headers -> check response
-  **Example** -  **User-Agent=plugin_name=Checkmarx One Vulnerability Integration;plugin_version=1.0.18, X-SNC-INTEGRATION-SOURCE=0583165cdbb9fdd0199b172913961908, Host=eu.ast.checkmarx.net**
- Test same thing for all integrations
